docs/artifactory/groupdelete/help.go

```diff
@@ -0,0 +1,8 @@
+package groupdelete
+
+const Description = "Delete a group from the JFrog Platform."
+
+var Usage = []string{`jfrog rt gdel <Group Name>`}
+
+const Arguments string = `	group name
+Specifies the desired name of the group to be deleted.`
```

Comment:
Specifies the desired name of the group to be deleted.`
-->
Group name to be deleted.`